config/locales/en.yml

```diff
@@ -2284,66 +2284,11 @@ en:
     structure:
       title: "Structure"
       board:
-        name: "WCA Board of Directors"
         description: "The WCA Board is responsible for leading the organization as a whole, and fulfilling any duties not fulfilled by other Teams,  Committees, and Councils"
       users:
         profile: "%{user_name}'s profile"
-      #context: WDC
-      wdc:
-        name: "WCA Disciplinary Committee"
-      #context: WDPC
-      wdpc:
-        name: "WCA Data Protection Committee"
-      #context: WRC
-      wrc:
-        name: "WCA Regulations Committee"
-      #context: WRT
-      wrt:
-        name: "WCA Results Team"
-      #context: WST
-      wst:
-        name: "WCA Software Team"
-      #context: WCT
-      wct:
-        name: "WCA Communication Team"
-      #context: WFC
-      wfc:
-        name: "WCA Financial Committee"
-      #context: WEC
-      wec:
-        name: "WCA Ethics Committee"
-      #context: WEAT
-      weat:
-        name: "WCA Executive Assistants Team"
-      #context: WQAC
-      wqac:
-        name: "WCA Quality Assurance Committee"
-      #context: WCAT
-      wcat:
-        name: "WCA Competition Announcement Team"
-      #context: WMT
-      wmt:
-        name: "WCA Marketing Team"
-      #context: WAC
-      wac:
-        name: "WCA Advisory Council"
-      #context: WSOT
-      wsot:
-        name: "WCA Sports Organization Team"
-      #context: WAT
-      wat:
-        name: "WCA Archive Team"
-      #context: Hidden team for list of banned competitors
-      banned:
-        name: "Banned Competitors"
-      #context: Hidden team for WCT China members
-      wct_china:
-        name: "WCT China"
```

Comment:
where did these go? Are there already adequate replacements?

Reply:
These were used to show in the teams page probably. They already have a replacement under `page.teams_committees_councils.groups_name`.